packages/perseus/src/renderer.tsx

```diff
@@ -1533,10 +1511,6 @@ class Renderer
             if (widget?.getDOMNodeForPath) {
                 return widget.getDOMNodeForPath(interWidgetPath);
             }
-            if (interWidgetPath.length === 0) {
-                // @ts-expect-error - TS2345 - Argument of type 'Widget | null | undefined' is not assignable to parameter of type 'ReactInstance | null | undefined'.
-                return ReactDOM.findDOMNode(widget);
-            }
```

Comment:
We only ever seem to hit line 1536 when we are in a Graded Group, but the length is never zero. I've been unable to find any cases where we ever actually need this fallback. I tested every widget in storybook in both desktop/mobile views. The only place we ever satisfy this condition is when we're explicitly forcing the situation in tests.